.gitignore

```diff
@@ -127,3 +127,10 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Ignore packagelist metadata DUMP
+scripts/ingestion/parsers/Packagelist_DUMP/20*
```

Comment:
if we are generating these files, it may be useful to use a custom extension (e.g., `.dump`) and then ignore it with a simpler pattern such as `**/*.dump`